examples/geostrophic_adjustment.jl

```diff
@@ -35,7 +36,9 @@ coriolis = FPlane(f=1e-4)
 #
 # We use `grid` and `coriolis` to build a simple `HydrostaticFreeSurfaceModel`,
 
-model = HydrostaticFreeSurfaceModel(grid=grid, coriolis=coriolis)
+model = HydrostaticFreeSurfaceModel(grid = grid,
+                                    coriolis = coriolis,
+                                    free_surface=ImplicitFreeSurface())
```

Comment:
Should the time step be updated now that the example using an implicit free surface? The time step is still chosen based on the `wave_propagation_time_scale` below.

Reply:
Well, the gravity waves are definitely wrong with a larger time step. So I guess it depends what you want the example to show --- a stable model that can take a huge time step, or a physically accurate geostrophic adjustment?